telegram/ext/_commandhandler.py

```diff
@@ -216,78 +237,84 @@ class PrefixHandler(CommandHandler):
             Defaults to :obj:`False`.
 
     Attributes:
+        prefixes (Set[:obj:`str`]):
+            The prefix(es) that will precede :attr:`commands`.
+        commands (Set[:obj:`str`]):
+            The set of command(s) that this handler should listen for.
         callback (:obj:`callable`): The callback function for this handler.
         filters (:class:`telegram.ext.BaseFilter`): Optional. Only allow updates with these
             Filters.
         run_async (:obj:`bool`): Determines whether the callback will run asynchronously.
 
+            .. versionchanged:: 14.0
+                :attr:`prefix` has been changed to :attr:`prefixes` and accepts sets as well.
+                :attr:`command` has been changed to :attr:`commands` and accepts sets as well.
```

Comment:
Please move the "changed to" notes to a `versionchanged` directive right before the `Args:`. Please move the "now accepts sets" to a `versionchanged` directive in the documentation of the respective arguments